actions/transactions/new.go

```diff
@@ -16,8 +16,7 @@ import (
 // @Description	New transaction
 // @Tags		Transactions
 // @Produce		json
-// @Param		config query string true "transaction config"
-// @Param		metadata query string false "metadata"
+// @Param		NewTransaction body NewTransaction true "NewTransaction model containing the transaction config and metadata"
```

Comment:
New Transaction does not have transaction config and metadata in swagger
<img width="1361" alt="transaction" src="https://github.com/bitcoin-sv/spv-wallet/assets/150656925/173da96d-8bcc-4b59-9cde-6a916fdcbd02">

Reply:
Done